### ISSUE TITLE
YOU SENT V2 OUT INTO THE PPAS!

### ISSUE THREAD

**Issue by tarjei** [influxdata/influxdb#20043]
Hi, you just ruined my saturday. 

I normaly try to be nice when creating bugreports but what you guys have done now is just plain disrespectfull and rude.

I will not even bother to spellcheck this post.

What on earth propelled you to just dump the 2.x version of influx into your PPAs? 

I can relate to you wanting to let your users use the latest version of influx and I wanted to upgrade, but not at random when I just wanted the latest bugfixes and securityfixes. 

So: Either you do not like people using influx through your ppa or you have done a huge misstake.

Either way: You should remove v2 from the PPAs and other repos at once and move them to separate urls so it is possible for system admins to actually decide when to upgrade to v2.

__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. ...
2. ...
3. ...

__Expected behavior:__
Describe what you expected to happen.

__Actual behavior:__
Describe What actually happened.

__Environment info:__

* System info: Run `uname -srm` and copy the output here
* InfluxDB version: Run `influxd version` and copy the output here
* Other relevant environment details: Container runtime, disk info, etc

__Config:__
Copy any non-default config values here or attach the full config as a gist or file.

__Logs:__
Include snippet of errors in log.

__Performance:__
Generate profiles with the following commands for bugs related to performance, locking, out of memory (OOM), etc.

```sh
# Commands should be run when the bug is actively happening.
# Note: This command will run for at least 30 seconds.
curl -o profiles.tar.gz "http://localhost:8086/debug/pprof/all?cpu=true"
curl -o vars.txt "http://localhost:8086/debug/vars"
iostat -xd 1 30 > iostat.txt
# Attach the `profiles.tar.gz`, `vars.txt`, and `iostat.txt` output files.
```

**Comment by tarjei** [comment]
It seems that you actually have fixed the ppa. I will therefore close this issue now.

**Comment by DragonQ** [comment]
This is actually rather annoying. My server auto-updates packages every night and one morning I woke up with a completely broken Influx instance. Turns out it had updated to v2.0.1 and I had to recreate a lot of stuff from scratch. I now have an old v1 database and a new v2 database, and am unsure how to merge the two. Any ideas on this?

Additionally, I'm stuck on 2.0.1 for the foreseeable future because 2.0.2 and the upcoming 2.0.3 are not being added to the repo. Since I want a feature that's going to be included in 2.0.3, I'm kinda stuck. Can you not have two stable repos, one for v1 and one for v2? That would solve a lot of these problems.
